benches/lib.rs

```diff
@@ -59,3 +59,12 @@ fn is_subset_4(b: &mut Bencher) {
     let sup: RoaringBitmap = (0..65536).collect();
     b.iter(|| test::black_box(sub.is_subset(&sup)))
 }
+
+#[bench]
+fn remove_range_bitmap(b: &mut Bencher) {
+    let mut sub: RoaringBitmap = (0..65536).collect();
+    b.iter(|| {
+        test::black_box(sub.remove_range(4096 + 1, 65536));
```

Comment:
Ah, I guess it's still running that code since the bitmap isn't fully removed, just not removing anything each time. Might pay to put a comment here about that.